### ISSUE TITLE
Make header & footer more minimal

### ISSUE THREAD

**Issue by chadoh** [citrusbyte/thicket-intro#4]
@s-small this is a conversation we've been having in other channels that I wanted to move here.

I talked to Lucas about this a bit, and he's about as unopinionated as I am between "make it match citrusbyte.com" and "make it its own thing." The reason I am leaning toward "make it its own thing" is because we do plan and hope to have many other minimal little pages built for other projects we build. If we go with "make it match citrusbyte.com," then all of these pages could easily become out-of-date with citrusbyte.com, when it changes in the future. He also sees no reason to think that we would even keep a header or footer at all, in future versions of citrusbyte.com, and so extracting them to shared components that can be used across our apps doesn't even make sense.

I wanted to run this by you, and see if 1. you agree and if so, 2. wanna design up a minimal header & footer for `thicket-intro`? I was thinking the header could be _just_ the Citrusbyte logo on the left, and the footer could be just the Citrusbyte logo and maybe the social links. But it's up to you, really, as you feel the strongest about this.

**Comment by s-small** [comment]
@chadoh Agreed with making it its own thing. Your rationale about dating totally makes sense. Totally down to design up a minimal header and footer for Thicket Intro. We may want to think a little more holistically in the future about the header logo being the CitrusLabs logo and any other header links that may be associated with CitrusLabs like "About" or other relevant links, but for the time being I'm cool with just the CB logo as the header. In the footer, I'm cool with a small circle CB logo and some social links. And we may want to think about any terms, privacy or other small footer links we want incorporated. 

I won't get to this until next week obviously, but let me know your thoughts and I'll make sure to note as a to do for early next week.

**Comment by chadoh** [comment]
@s-small yes, those sound good. I agree that long-term we will want some reference to CitrusLabs, but for now we are moving away from explicitly talking about it. Will wants to make sure that Labs is a real thing before we start referencing it.
